datasets/Berk_2021/README.md

```diff
@@ -0,0 +1,6 @@
+> Berk, K. A. C., van der Voorn, B., & van Rossum, E. F. C. (2021). [Intermitterend vasten in wetenschap en praktijk][1]. Nederlands Tijdschrift voor Geneeskunde, 165.
+
+Processes 1 [raw dataset][2]. Only processed first and second search, as the third search was screened using ASReview and should therefore not be used for benchmarking.
```

Comment:
How does this work? Are these searches different datasets?

Reply:
@J535D165, the [description of the dataset on OSF](https://osf.io/9x6ca/wiki/home/) describes it as follows:

> To increase the quality of a systematic review on Intermittent Fasting, we applied a 2-step screening. First, a classical search and screening process was performed identifying 638 records of which 18 were deemed as relevant. 

This is search 1 and 2 that I processed in the script.

> Then, the search was updated with identifying 53 new publications. The labels of the first search were used as training data for the second search using ASReview. Then, the 53 records were screened and 7 additionally relevant records were identified.

So if all 53 records were screened, it could be included for benchmarking, however this is not explicitly mentioned, only that ASReview was used to screen those 53 records from search 3.